lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "layerEditorCommand.h"
 
+#include <mayaUsd/ufe/Utils.h>
```

Comment:
As we discussed yesterday all the changes in this file need to be UFE protected, with something like:
#if defined(WANT_UFE_BUILD)

Reply:
Yeah, I'm on it.

Reply:
As per our coding guidelines all conditional includes should come last (after the system includes).